docs/spec/governance/state.md

```diff
@@ -194,7 +194,7 @@ And the pseudocode for the `ProposalProcessingQueue`:
 
       // Slash validators that did not vote, or update tally if they voted
       for each validator in validators
-        if (validator.bondHeight < CurrentBlock - tallyingProcedure.GracePeriod)
+        if (validator.bondTime < CurrentTime - tallyingProcedure.GracePeriod)
```

Comment:
Do we want to use `time.Before` and `time.Sub` here?

Reply:
Same here.  Use those in the code.  This was just pseudocode in the docs.